src/Pyz/Yves/ShopUi/Theme/default/components/organisms/header/header.twig

```diff
@@ -146,7 +146,7 @@
     </div>
 
     {% if data.showNavigation %}
-        <div class="{{ config.name }}__navigation">
+        <div class="{{ config.name }}__navigation" id="main-navigation">
```

Comment:
where from we know that we are using `main-navigation` from somewhere?

Reply:
With variables will be changed with the attribute